test/integration/api/projects.js

```diff
@@ -1180,3 +1180,51 @@ describe('api: /projects', () => {
   });
 });
 
+describe('api: /projects?forms=true', () => {
+  describe('GET', () => {
+    it('should return projects with nested extended forms', testService((service) =>
+      service.login('alice', (asAlice) => asAlice.get('/v1/projects?forms=true')
+        .expect(200)
+        .then(({ body }) => {
+          body.length.should.equal(1);
+          body[0].should.be.a.Project();
+          const form = body[0].nestedForms[0];
+          form.should.be.a.Form();
```

Comment:
There's also an `ExtendedForm()` assertion. Actually, probably a good idea to update that assertion to incorporate the new `reviewState` property.